docs/creating_integration_manifest.md

```diff
@@ -28,7 +29,8 @@ Or a minimal example that you can copy into your project:
   "documentation": "https://www.example.com",
   "dependencies": [],
   "codeowners": [],
-  "requirements": []
+  "requirements": [],
+  "iot_class": "cloud_polling"
```

Comment:
IoT class is optional so can be left out. Actually the minimal example could be just `domain` and `name` maybe 🤔

Reply:
IoT class is not optional...

Reply:
Ok!